.github/workflows/wheels.yml

```
@@ -0,0 +1,43 @@
+name: Build-wheels
+on:
+  release:
+  workflow_dispatch:
+  
+jobs:
+  build_wheels:
```

Similarly, we (arbitrarily) prefer hyphens to underscores.
```suggestion
  build-wheels:
```